scripts/daily_upload_s3.coffee

```diff
@@ -21,25 +20,30 @@ switch process.env.NODE_ENV
 # Connect to database
 db = mongojs(process.env.MONGOHQ_URL, ['articles'])
 
-# Setup CSV
+# Setup file naming
 filename = "export_" + moment().format('YYYYMMDDhhmmss') + ".csv"
-csvStream = csv.createWriteStream { headers: true }
 dir = 'scripts/tmp/'
 
-writableStream = fs.createWriteStream( dir + filename)
+projections = { 'id': 1, 'author_id': 1, 'auction_ids': 1, 'contributing_authors': 1, 'fair_ids': 1, 'featured': 1, 'featured_artist_ids': 1, 'featured_artwork_ids': 1, 'partner_ids': 1, 'primary_featured_artist_ids': 1, 'slugs': 1, 'tags': 1, 'title': 1, 'tier': 1, 'published_at': 1, 'show_ids': 1, 'section_ids': 1, 'thumbnail_image': 1, 'thumbnail_title': 1, 'keywords': 1, 'slug': 1, 'channel_id': 1, 'partner_channel_id': 1 }
```

Comment:
It might be nicer to do this in two steps so you can keep the list of attrs clean, e.g.

````coffeescript
attrs = ['id', 'author_id', ....]
projections = _.object attrs, attrs.map -> 1
````